repos.yml

```diff
@@ -1,15 +1,22 @@
 hr-attendance:
-    defaults:
-        depth: 1
     remotes:
         oca: https://github.com/OCA/hr-attendance.git
     merges:
         - oca 15.0
+        # https://github.com/OCA/hr-attendance/pull/140
+        - oca refs/pull/140/head
 hr-holidays:
+    remotes:
+        oca: https://github.com/OCA/hr-holidays.git
+    merges:
+        - oca 15.0
+        # https://github.com/OCA/hr-holidays/pull/87
+        - oca refs/pull/87/head
```

Comment:
@albig I record the merges I need for things to function here anyways, is this sufficient for you or would you like to have a flat list of PRs to merge somewhere else?

Reply:
That's ok for me.